lib/galaxy/webapps/galaxy/api/datatypes.py

```diff
@@ -111,15 +120,21 @@ async def converters(self) -> DatatypeConverterList:
         summary="Returns a dictionary/map of datatypes and EDAM formats",
         response_description="Dictionary/map of datatypes and EDAM formats",
     )
-    async def edam_formats(self) -> Dict[str, str]:
+    async def edam_formats(
+        self,
+        id_only: Optional[bool] = IdentifierOnly,
```

Comment:
I guess we're back at the question, should we have an extra path, a view parameter, or just a simple boolean here. What do you think about https://github.com/galaxyproject/galaxy/commit/4e8a9cb13b19b0cfc4ae91bf72108654b65c55df as an alternative ?

Reply:
I've thought a bit about it, and think the route option is preferable, when the response body changes structure.

Either way, `detailed` is more descriptive than `id_only`